add_model_instructions.md

```diff
@@ -0,0 +1,35 @@
+## Instruction of adding a model to trained_model repository
+
+1. we suggest to create a separate environment to submit your model.
+- install and configure [git](https://github.com/git-guides/install-git), [git-annex](https://git-annex.branchable.com/install/) and also you need a github account.
+- Install and configure [datalad](https://handbook.datalad.org/en/latest/intro/installation.html)
+- install [datalad-osf](https://handbook.datalad.org/en/latest/intro/installation.html) and get your credentials.
+
+2. Fork and then clone the trained_model repository from your Fork. You can use either git or datalad.
+
+	Using git:```git clone git@github.com:<your_github>/trained-models.git```
```

Comment:
since we are using datalad-osf it does not need to enable remote. all this stuff has been done under the hood which makes the process easier.
In general, I will go toward creating a github action as we discussed previously. but I needed someone else to test this process. I will change the subject to WIP.